### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by fullstop125**
**Portfolio: add a contact form and finish the mobile version**
In this section I managed to : 
- Create forms with HTML5 validations.
- Understand the importance of UX.
- Understand how to parse a Figma design to create a UI.
- Add a contact form to my portfolio and the portfolio of my coding partner.
- Make sure that I guide my users' experience in a way that helps them fill in the form without frustration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,276 +1,443 @@
...
-               <a href="#">
-                  <img class="social-media-icons" src="./images/icons/linkedin.png" alt="LinkedIn Icon">
-               </a>
+              <input
+                class="username"
+                type="text"
+                name="username"
+                placeholder="username"
+                required
+                maxlength="30"
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [ ] Great job 👍 . Kindly make sure your `input` tag isn't wrapped in a `ul` tag. 🙏 . It is best practice if they are just wrapped in a `form` tag

File: styles.css [hunk 2]
```diff
@@ -277,3 +276,80 @@ ul {
...
+form {
+  margin-right: 45px;
+  display: flex;
+  flex-direction: column;
+}
+
+.username {
+  width: 100%;
+  height: 48px;
+  padding: 5px;
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [x] Great job with your design 👍 . I can see you and your partner put in a lot of effort. Kindly make sure your design looks as close as possible to the Figma design as it is best practice. This issue is caused by the padding in which you used on your `main` tag.

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @fullstop125  and @elfin-git , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are passing ✔️ 
- Great Pull request description 💯 
- Great try with your design 👏 


## Required Changes ♻️

- [x] Kindly remove all unnecessary sections in your readme file so it looks more professional 🙏 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (fullstop125) [reply index.html]: thank you for the feedback , it's now corrected

**Review by Ismailco — APPROVED** [review]
### Hi @fullstop125, @elfin-git,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**